xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -484,20 +484,29 @@ void CDVDInputStreamBluray::ProcessEvent() {
   case BD_EVENT_TITLE:
   {
     CLog::Log(LOGDEBUG, "CDVDInputStreamBluray - BD_EVENT_TITLE %d", m_event.param);
+    m_player->OnDiscNavResult(nullptr, BD_EVENT_TITLE);
     const BLURAY_DISC_INFO* disc_info = bd_get_disc_info(m_bd);
 
     if (m_event.param == BLURAY_TITLE_TOP_MENU)
     {
       m_title = disc_info->top_menu;
+      m_player->OnDiscNavResult(nullptr, BLURAY_TITLE_TOP_MENU);
       m_menu = true;
       break;
     }
     else if (m_event.param == BLURAY_TITLE_FIRST_PLAY)
+    {
       m_title = disc_info->first_play;
+      m_player->OnDiscNavResult(nullptr, BLURAY_TITLE_FIRST_PLAY);
+    }
     else if (m_event.param <= disc_info->num_titles)
       m_title = disc_info->titles[m_event.param];
     else
       m_title = nullptr;
+  
+    if (m_event.param == BLURAY_UO_TITLE_SEARCH)
+      m_player->OnDiscNavResult(nullptr, BLURAY_UO_TITLE_SEARCH);
```

Comment:
isn't this superflouos since what this event does in VideoPlayer is to set `IsBlurayTopMenu` to false, which you already do at the beginning of this case in the event `BD_EVENT_TITLE` which is fired right before.

Reply:
The event bluray user operation title search can be triggered when a user uses inner menus during movie playback to change chapter or just change the position in the time bar. When this happens, playback will continue and it should be set to false, because we are not inside the top menu of a disc. 

This is passing the event to videoplayer so it can use IsBlurayTopMenu.
BD_EVENT_TITLE might happen but in some discs only bluray user operation might be triggered. 

I don't have any disc that this happens but this is more of a safe measure for all discs to work properly, because the scenario might happen. 
It can be removed but it would be a more safer bet to leave it in.

Reply:
you are firing this event from within the `BD_EVENT_TITLE ` event case, so it will always happen before you forward this search event to videoplayer. Thus, these lines are superfluous. Or am I seeing this wrong?

Reply:
Ok, I'll remove those.